collectors/feature/jira/src/main/java/com/capitalone/dashboard/client/team/TeamDataClientImpl.java

```diff
@@ -63,13 +60,6 @@ public int updateTeamInformation() {
 		if (CollectionUtils.isNotEmpty(teams)) {
 			updateMongoInfo(teams);
 			count += teams.size();
-		} else {
-			List<BasicProject> projects = jiraClient.getProjects();
-			
-			if (CollectionUtils.isNotEmpty(projects)) {
-				updateMongoInfoLegacy(projects);
-				count += projects.size();
-			}
```

Comment:
@MarkRx @tabladrum I've made the necessary changes to refactor the Jira and VersionOne collectors. The change to collect features on-demand will be made after we merge the first set of changes.

Can you take a look at my fork and pull the code down and try it out before I send in the PR. The code that I've pushed to my fork includes parts of PR-1030 coupled with the re-factoring, so essentially it will replace PR-1030.  Once we review and merge the code from my fork, we will need to change PR-1054 as needed and merge it.

Steps to re-run the jira / versionone collectors:
1. Make a backup of your database.
2. delete collector_items where collectorId in (select collectors.id from collectors where collectors.name in ("Jira", "VersionOne") )
3. Drop feature , scope , team
4. Run jira and versionone collector
5. Open your dashboard and re-configure your feature widget to pick the right project / team value.

Script for Steps 2 and 3 is given below.

```
db.collectors.find({
	"name" : { $in : ["Jira", "VersionOne"] }   
}).forEach(
	function(myDoc) {
		db.collector_items.remove( {"collectorId" : myDoc._id} );
	}
);

print('Dropping feature collection');
db.feature.drop();

print('Dropping scope collection');
db.scope.drop();

print('Dropping team collection');
db.team.drop();
```